src/main/java/org/myrobotlab/service/Arduino.java

```diff
@@ -66,6 +66,8 @@
 import org.myrobotlab.service.interfaces.UltrasonicSensorControl;
 import org.myrobotlab.service.interfaces.UltrasonicSensorController;
 
+import akka.event.AddressTerminatedTopic;

```

Comment:
What is this import needed for?

Reply:
long gone .. dunno how that got in there ..